micro-rdk/src/common/ota.rs

```diff
@@ -364,14 +386,20 @@ impl<S: OtaMetadataStorage> OtaService<S> {
         let mut total_downloaded: usize = 0;
         let mut got_info = false;
 
+        log::info!("writing new firmware to address `{:#x}`", self.address,);
+
         while let Some(next) = response.frame().await {
-            let frame = next.unwrap();
+            // hyper::Error(Body, Error { kind: Io(Custom { kind: Other, error: "ssl other error -26624" })
```

Comment:
is this an accident?

Reply:
ah, yeah. was notes to myself for when I implement [RSDK-9214](https://viam.atlassian.net/browse/RSDK-9214?atlOrigin=eyJpIjoiYWVmYTU0YzI1OGRhNDNkMGEwNWU2YTMyZmIwMDc5ZDYiLCJwIjoiaiJ9), the final code-based ticket to improve the error messaging. will remove it before merging

Reply:
Please remember to remove before commit